### ISSUE TITLE
Build error on windows 10

### ISSUE THREAD

**Issue by axklim** [yiisoft/yii2-docker#26]
On windows 10 have a trouble:
```
$ docker-compose build
ERROR: .IOError: [Errno 2] No such file or directory: u'.\\docker-compose.yml:docker-compose.apache.yml'
```

**Comment by axklim** [comment]
Trouve solution:
add to `.env` file: `COMPOSE_PATH_SEPARATOR=:`
May be add to `.env-dist` line:

```
# Environment variables for docker-compose command

## Uncoment for Windows 10
# COMPOSE_PATH_SEPARATOR=:
...
```

**Comment by axklim** [comment]
#27

**Comment by schmunk42** [comment]
fixed in #27
